nmigen_boards/dev/spi.py

```diff
@@ -4,16 +4,20 @@
 __all__ = ["SPIResource"]
 
 
-def SPIResource(number, *, cs, clk, mosi, miso, int=None, reset=None, attrs=None):
```

Comment:
This breaks the signature of the function and every use of it in every board.

Reply:
Ahhh okay I see now. Although `SPIResource` is currently unused:

```
William@William-THINK MINGW64 ~/src/nmigen-boards
$ git grep -F "SPIResource" master
master:nmigen_boards/dev/__init__.py:from .spi import SPIResource
master:nmigen_boards/dev/spi.py:__all__ = ["SPIResource"]
master:nmigen_boards/dev/spi.py:def SPIResource(number, *, cs, clk, mosi, miso, int=None, reset=None, attrs=None):
```

Every `*Resource` function you have in `nmigen-boards` starts with a mandatory `number` argument. I still feel like `name` should be mandatory in case a user forgets to put it?